jina/peapods/zmq/__init__.py

```diff
@@ -495,7 +499,11 @@ def send_message(
     num_bytes = 0
     try:
         _prep_send_socket(sock, timeout)
-        sock.send_multipart(msg.dump())
+        if 'flags' in kwargs:
```

Comment:
if we ever receive flags in kwargs, lets put it as Optional parametrr with default to None

Reply:
Not really easy to do, since the kwargs are passed as part of other kwargs, from up the call stack